.gitignore

```diff
@@ -11,4 +11,6 @@ stacks/*
 environment.sh
 
 # editor config stuff
+.idea
```

Comment:
was this part of your working this stack? If so should it be in your personal gitignore rather than here if there's no chance of it coming up again?

Reply:
I don't know my idea just keeps adding it. Ermmmmm that is consideration. I may need to set this up though.